src/napari_live_recording/processing_engine_/image_filters/__init__.py

```diff
@@ -0,0 +1,28 @@
+from os.path import dirname, basename, isfile, join
+import glob
+
+
+class FilterFunction:
+    def __init__(self, parameters) -> None:
+        self.parameters = parameters
+        pass
+
+
+class Filter:
+    def __init__(self, filter: "function") -> None:
+        self.filter = filter
+
+    def executeFilter(self, image, parameterDict):
```

Comment:
you could change this with the `__call__` method.

Basically whenever you create an object, i.e.

```py
class MyObject:
    def __call__(self, msg):
         print(f`hello there, {msg}`)
         
obj = MyObject()
MyObject("General Kenobi")
```

using the `__call__` function allows you to use the syntax above